app/controllers/submit.js

```diff
@@ -720,6 +720,12 @@ export default Ember.Controller.extend(Analytics, BasicsValidations, NodeActions
                     },
                     license: this.get('basicsLicense.licenseType')
                 });
+                node.setProperties({
+                    description: currentAbstract,
+                    tags: currentTags,
+                    license: this.get('basicsLicense.licenseType'),
+                    nodeLicense: this.get('basicsLicense.licenseType'),
+                });
```

Comment:
This should only happen if the check box to apply the license to the node is checked.